hubspot/tasks.py

```diff
@@ -51,6 +77,20 @@ def check_hubspot_api_errors():
             error_type = error.get("type", "N/A")
             details = error.get("details", "")
 
+            if (
+                obj_id is not None and
+                "LINE_ITEM" in obj_type and
+                error_type == "INVALID_ASSOCIATION_PROPERTY" and
+                ASSOCIATED_DEAL_RE.search(details) is not None
+            ):
+                try:
+                    personal_price = PersonalPrice.objects.get(id=obj_id)
+                except ObjectDoesNotExist:
+                    pass
```

Comment:
nit: add a `log.error` here so that we can know if this ever happens, even if it shouldn't ever.